public/app/plugins/datasource/prometheus/configuration/ExemplarSetting.tsx

```diff
@@ -74,12 +74,37 @@ export default function ExemplarSetting({ value, onChange, onDelete }: Props) {
                 datasourceUid: undefined,
                 name: value.name,
                 url: event.currentTarget.value,
+                urlDisplayLabel: value.urlDisplayLabel,
               })
             }
           />
         </InlineField>
       )}
 
+      {isInternalLink ? (
+        ''
+      ) : (
+        <InlineField
+          label="URL Label"
+          labelWidth={24}
+          tooltip="Use to override the label of the URL button for the externel link."
+        >
+          <Input
+            placeholder="Go to example.com"
+            spellCheck={false}
+            width={40}
+            value={value.urlDisplayLabel}
+            onChange={(event) =>
+              onChange({
+                datasourceUid: undefined,
```

Comment:
Maybe a bit nitpicky, but you can destructure the current value instead of assigning each value like on L120. 

`onChange({ ...value, urlDisplayLabel: event.currentTarget.value })`